backend/utils/volunteer-sheet-parser.js

```diff
@@ -0,0 +1,87 @@
+// run `node utils/volunteer-sheet-parser.js <insert_spreadsheet_path>` from the `/backend` folder
+
+const XLSX = require("xlsx");
+const extractor = require("keyword-extractor");
+const resourceUtils = require("./resource-utils");
+const Resource = require("../models/Resource");
+const mongoose = require("mongoose");
+
+mongoose.connect(process.env.DB_URI, {
+  useUnifiedTopology: true,
+  useNewUrlParser: true
+});
+
+const formatNotes = ({
+  "Form Received": formReceived,
+  Availability: availability,

```

Comment:
Let's keep consistency if we're using strings in object notation.

Reply:
Lol this is because of prettier I think. Since there's no space in the string it removes the quotes :(. When I try adding them in, it fails the prettier check.